MapPointMapper/ViewController.swift

```diff
@@ -108,14 +109,31 @@ class ViewController: NSViewController, MKMapViewDelegate, NSTextFieldDelegate {
     renderer.strokeColor = colorWell.color
     return renderer
   }
-  
-  // MARK: NSTextFieldDelegate
-  override func keyUp(theEvent: NSEvent) {
```

Comment:
Do we want to keep the `keyUp` hack and just get the current `firstResponder` to know which field/button to 'press'?